packages/rum-core/src/domain/rumEventsCollection/longTask/longTaskCollection.ts

```diff
@@ -19,7 +19,7 @@ export function startLongTaskCollection(lifeCycle: LifeCycle) {
     lifeCycle.notify(LifeCycleEventType.RAW_RUM_EVENT_COLLECTED, {
       rawRumEvent,
       startTime: startClocks.relative,
-      domainContext: {},
+      domainContext: { performanceEntry: (entry as unknown) as PerformanceEntry },
```

Comment:
Is there a way to avoid this cast?

Reply:
See my answer there: https://github.com/DataDog/browser-sdk/pull/883#discussion_r646412570
We could if we exposed the `RumPerformanceEntry` types publicly.